frontend/src/components/tabs/TabLink.tsx

```diff
@@ -0,0 +1,24 @@
+import cx from 'classnames'
+import { FunctionComponent } from 'react'
+import { Link, LinkProps, useLocation } from 'react-router-dom'
+
+const TabLink: FunctionComponent<LinkProps> = ({ children, ...otherProps }) => {
```

Comment:
Added the `TabLink` and `TabList` components:

<img width="339" alt="Screen Shot 2021-03-19 at 3 36 48 PM" src="https://user-images.githubusercontent.com/3411183/111848695-f0d10900-88c8-11eb-925a-a08158742a2b.png">

Reply:
ah nice!